dotcom-rendering/src/web/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -195,14 +195,15 @@ const columnStyle = css`
 export const Column = ({
 	column,
 	index,
+	showLine,
 }: {
 	column: PillarType;
 	index: number;
+	showLine: boolean;
```

Comment:
How about `isLastColumn`? I find it helps if props are declarative, rather than imperative. You can google this stuff but essentially the thinking goes that it's easier to scale code if your props are declaring the state of things and then leaving it up to the component to decide what to do rather than being imperative and telling the component to do a thing